.github/workflows/percy_snapshots.yml

```diff
@@ -40,6 +42,8 @@ jobs:
           node-version: 16.x
       - run: npm ci
       - run: ${{ inputs.build_script }}
+        env:
+          REACT_APP_MAPBOX_API_KEY: ${{ secrets.REACT_APP_MAPBOX_API_KEY }}
```

Comment:
For my own edification, why does the build script need the API key?

Reply:
Because the other step occurs after the storybook is built. Which is where it reads env vars from node's process and copies them into the "fake" storybook process